var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -84,7 +85,12 @@ def determine_version(cls, lib):
     @property
     def libs(self):
         shared = "+shared" in self.spec
-        return find_libraries(["libz"], root=self.prefix, recursive=True, shared=shared)
+        libnames = ["libz"]
+        if sys.platform == "win32":
```

Comment:
A couple points:

* (question) Is it worth searching for `libz` on windows?
* Can you refactor this to `if self.spec.satisfies(platform=windows)`?

Reply:
For externals that makes sense